src/SearchResults.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import "./css/Footer.css";
+import FakeBookingData from "./data/fakeBookings.json";
```

Comment:
If you read the instructions in exercise 6 carefully it says that you should pass the fake bookings data from the `Bookings` component as a prop to the `SearchResults` component. Can you fix that please?

Reply:
exercise 6 issues have resolved